blinds/blinds_api.py

```diff
@@ -78,6 +92,16 @@ class SmartBlindsSystem:
     _blindsSchedule = None
     _temperatureSensor = None
 
+    '''
+    Costructor for modelling the system of blinds as a whole. 
+    Provides functions for API requests
+
+    Arguments:
+        blinds : a Blinds object for controlling the blinds themselves. This is currently left as a single object for proof 
+            concept, but can be extended to a list of Blinds objects for controlling multiple blinds throughout the house
+        blindsSchedule : a BlidnsSchedule object to control the schedule of the blinds
+        temperatureHandler : an abstraction of the temperature sensor controls 
```

Comment:
I changed the variable name from `temperatureHandler` to `temperatureSensor`. Can you update the comment?

Reply:
Fixed